CONTRIBUTING.md

```diff
@@ -0,0 +1,231 @@
+- If it's a serious bug we might ask you for permission to take over your branch and continue the work ourselves.
+- Unfortunately we often see contributors submitting PR's, correcting a few of our suggested changes but then disappear. Please don't do that! If you realize you won't find the time at all to continue your work, **please just let us know**. It's perfectly fine and there won't be hard feelings. If we consider your feature useful we might even take over and finish it ourselves.
+- We try to release merged features as soon as possible. Even the tiniest feature deserves a release. Sometimes we just can't make it soon enough because we don't find the time or we wait for something else because it makes sense to be released together. If a feature you submitted is merged and you'd need it urgently to be available on PyPi, **please kindly remind us** and we'll try to find the time.
+
+
+## Discuss & Design First
+
+It proved to be useful in the past if loose (feature) ideas would be discussed in #synadm:peek-a-boo.at first, which helps forming a more concrete idea that can than further be summarized into a well written github issue and finally be implemented by a `synadm` contributor or a `synadm` maintainer.
+
+- If you'd like someone else to pick up your feature idea because you are not able to code it yourself, open a detailed GitHub issue describing the feature.
+- If you'd like to code it yourself, a separate issue is not required, just describe your feature within the PR.
+
+
+## Getting the Source & Installing
+
+### Install into a Virtual Python Environment
+
+To install `synadm` into its own private virtual Python environment, FIXME
+
+<!-- omit in toc -->
+#### 1. Check Python Version
+
+`python3 --version` should show at least v3.6.x
+
+<!-- omit in toc -->
+#### 2. Clone Repo:
+
+```
+git clone https://github.com/joj0/synadm
+```
+
+<!-- omit in toc -->
+#### 3. Set up a Python3 Virtual Environment
+
+
+Create and activate a virtual environment using the python3 venv module:
+
+```
+python3 -m venv ~/.venvs/synadm
+source ~/.venvs/synadm/bin/activate
+```
+
+Once your Python virtual environment is loaded and your prompt looks similar to `(synadm) ....  $ `, install directly into the environment:
+
+
+<!-- omit in toc -->
+#### 4. Install
+
+```
+cd synadm
+pip install .
+```
+
+*Note: Don't forget to activate the venv when coming back to using `synadm` after a fresh login: `source ~/.venvs/synadm/bin/activate`*
+
+<!-- omit in toc -->
+#### 5. Run
+
+As long as your venv is loaded `synadm` should run fine without having to add a path in front of it
+
+```
+synadm -h
+```
+
+
+### Install in Editable Mode
+
+To be able to instantly be able to test the code you've been working on it makes sense to install from your git clone in _editable mode_
```

Comment:
```suggestion
To be able to instantly be able to test the code you've been working on it makes sense to install from your local clone of `synadm` in _editable mode_
```

Reply:
Yeah I tried to make that clear here that it's for editing/developing